src/components/studies/scheduler/read-only-pages/ReadOnlyAssessmentWindow.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+import {Box, makeStyles} from '@material-ui/core'
+import {latoFont} from '../../../../style/theme'
+import ClockIcon from '../../../../assets/clock.svg'
+import moment from 'moment'
+import {getTimeUnitFormatted} from '../utility'
+
+const useStyles = makeStyles(theme => ({
+  container: {
+    backgroundColor: 'white',
+    padding: theme.spacing(2),
+    marginBottom: theme.spacing(2),
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    height: '164px',
+  },
+  topLevel: {
+    display: 'flex',
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    width: '100%',
+    marginBottom: theme.spacing(3),
+    fontFamily: latoFont,
+    fontSize: '16px',
+  },
+  timesContainer: {
+    display: 'flex',
+    flexDirection: 'column',
+    width: '190px',
+    fontSize: '16px',
+  },
+  row: {
```

Comment:
This class is duplicated in a few places. Please add it to scheduler common styles and import it